k8s/bff/config.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rms-bff-config
+  namespace: rms
+  labels:
+    app: rms-bff
+data:
+  DB_HOST: "postgres" # Deve refletir o nome do service do PostgreSQL
+  DB_PORT: "5432"
+  DB_USERNAME: "pguser"
+  DB_PASSWORD: "pgpwd"
```

Comment:
não tem uma forma de pegar por variavel de ambiente?
pensando no mundo real...isso não poderia ser exposto.

Reply:
Então @anaplopes, cada linha em baixo de data vai ser transformada em uma variável de ambiente dentro do pod pelo kubernetes na hora de executar o container. Na prática no mercado a segurança do ConfigMap é garantida pelo RBAC do Kubernetes, então na prática só quem tem perfil e acesso de ADM (geralmente os LTs, os Tech Managers e os arquitetos) consegue abrir e ver os valores dentro do ConfigMap. Mas, na maioria das vezes, o pessoal linka o kubernetes com o Key Vault do Azure, ou com o KMS da AWS ou do Google e o kubernetes puxa esses valores das variáveis de ambiente de lá. Aí o controle de acesso as variáveis de ambiente fica a cargo do provedor de nuvem.

Reply:
Aqui no nosso cenário fomos pelo mais simples, mas na prática no mercado não fica assim aberto.

Reply:
Se vc esta seguro que não vai prejudicar nossa nota...por mim tudo bem.
só comentei pq credenciais expostas assim é uma falha de segurança.

Reply:
Opa, sim @anaplopes fez bem. Podemos tentar colocar as variáveis de ambiente no KMS da AWS depois, quando a gente for levar pra nuvem.